scss/components/_table-styles.scss

```diff
@@ -316,10 +316,25 @@ h3 + .simple-table {
 
 // Dense table
 // For tables with a lot of data that need to be condensed
+.dense-table-container {
+  box-shadow: -10px 0 20px -10px $gray-medium inset;
+  overflow-x: scroll;
+
+  @include media($lg) {
+    box-shadow: none;
+  }
+}
+
+.option__content .dense-table-container {
+  box-shadow: -10px 0 20px -10px $gray-medium inset;
```

Comment:
Good question, I wanted to doublecheck this with you...So when the table is inside 
"option__content", it is in between left and right sidebars (sticky-nav and aside, respectively) so it gets x-scrolling and shadow visual cue. Otherwise when in any larger container, like "main__content--right", it only gets x-scrolling and shadow when the screen itself is small. 
I dont know how this works exactly but when you add an aside to a section, it changes the wraps the original section in"option__content", which is perfect for targeting just this situation. Does that make sense? We can pair on this is if it is confusing.
I also wanted to ask if you think there are other places than resource block that this needs to be in.